src/payment-agents/PaymentAgentsItem.js

```diff
@@ -0,0 +1,33 @@
+import React, { PropTypes } from 'react';
+import { M, NumberPlain } from '../_common';
+
+const PaymentAgentsItem = ({ paymentAgent }) => {
+    return (
+        <tr>
+            <td>{paymentAgent.name}</td>
+            <td>
+                <div>
+                    <M m="Deposit commision: " />
+                    <NumberPlain
+                        currency={paymentAgent.currencies}
+                        value={paymentAgent.deposit_commission} />
+                </div>
+                <div>
+                    <M m="Withdrawal commision: " />
+                    <NumberPlain
+                        currency={paymentAgent.currencies}
+                        value={paymentAgent.withdrawal_commission} />
+                </div>
+            </td>
+            <td>
+                Available banks: ...
```

Comment:
We'll show all the info somehow, but we'll consider a better design than what we have for binary.com
For mobile - possibly a full page with the info, but nothing for now.